### PR TITLE
Maint/master/updates

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: file_paths.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ The files annotated by an '*' indicate that they are created by package installa
         mco@ *                            -> /opt/puppetlabs/puppet/bin/mco
         puppet@ *                         -> /opt/puppetlabs/puppet/bin/puppet
 
-    /opt/puppetlabs/facter
-        facts.d                           # external facts directory (not pluginsync'ed)
+    /opt/puppetlabs/facter *
+        facts.d *                         # external facts directory (not pluginsync'ed)
```
Comment (joshcooper) [inline file_paths.md]: Do we create this on windows too? should we? /cc @Iristyle

**Comment by MikaelSmith** [comment]
:+1:

**Comment by er0ck** [comment]
@melissa @joshcooper could this PR contain a commit to reflect the update to hieradata pathing?

**Comment by melissa** [comment]
@er0ck see https://github.com/melissa/puppet-specifications/commit/bb6751a3c2fdb3f48ace4c40f2859a605f227b30

File: file_paths.md (hunk 1 above)
Comment (Iristyle) [inline file_paths.md]: Yes, we already create this on Windows at `C:\ProgramData\PuppetLabs\facter\facts.d`

https://github.com/puppetlabs/puppet-specifications/blame/master/file_paths.md#L152-L153

**Comment by joshcooper** [comment]
:+1:

**Pull request merged**